packages/kit/src/runtime/server/parse_body/index.js

```diff
@@ -1,31 +1,34 @@
 import { read_only_form_data } from './read_only_form_data.js';
 
 /**
- * @param {import('types/hooks').StrictBody} raw
- * @param {import('types/helper').Headers} headers
+ * @param {Uint8Array} raw
+ * @param {Partial<import('types/helper').Headers>} headers
```

Comment:
Perhaps we can revisit #1611 if this is indeed the case for incoming headers, @benmccann?

Reply:
I thought it would be safer to assume headers were optional, since that could have prevented #1911 (`headers['content-type']` was `undefined`)

Reply:
What does `Partial<Record<string, string>>` even mean? Surely it's the same as `Record<string, string>`?

Reply:
It means the same as `Record<string, string | undefined>`

Reply:
Interesting, thanks. I think we probably just want the `Headers` type in that case, no? If a property exists on that object, it should be a string (if it's undefined, it's a bug)

Reply:
Ok